website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -784,9 +792,9 @@ The refined ORCA inventory catalog build script is available for download <a hre
 ### Addressing open questions
 
 - What is the proper date to use when receiving a date range from Cumulus for a reconciliation report?
-  - At this time, we are not going to include granule acquisition date since we cannot get it natively from cumulus but might add it in the future.
+  - After discussion with the Cumulus team, we have decided to use `cumulus_createdAt` timestamp.
```

Comment:
Update this sentence. Should explain that we are using the `createdAt` value passed by Cumulus in the CMA Message that will contain the timestamp that the granule began ingest of the granule into Cumulus. This date/time value will be used to filter results for Cumulus win asking for ORCA catalog inventory information. If you want you can further explain that it will be captured in the Cumulus granule table column.